_posts/2016-08-28-mouse-dna-magnetic-fields.md

```diff
@@ -0,0 +1,49 @@
+However, it is thought that prolonged exposure to ambient electromagnetic (EM) fields can affect certain cells in the body, causing a variety of cancers.
+For this reason, several countries have adopted limits for EM exposure, with most European countries limiting flux densities to 0.1&nbsp;mT ([milliteslas](https://en.wikipedia.org/wiki/Tesla_(unit))).
+
+To test the effects of EM fields on biological cells in a controlled environment, researchers exposed mice to low-frequency fluxes of 0.1&nbsp;mT and 1.0&nbsp;mT over eight weeks, with control mice living in similar conditions but without the EM exposure.
+The frequency chosen for the research was 50&nbsp;Hz, which is the frequency of transmission for alternating current in most countries.
+[This work](http://dx.doi.org/10.1371/journal.pone.0109774) is a follow-up to the same group's study of the effects of an eight-week exposure to a flux of 1.5&nbsp;mT, which is considerably higher than the normal limits and which showed unrepaired damage to DNA.
+In both these studies, the scientists were looking for signs of so-called unrepaired single-strand breakage in nuclear DNA.
+In the latest study, they also looked for indications that damaged DNA had been repaired and for signs that mitochondrial DNA was being produced at lower rates than usual.
+
+<!--break-->
+
+Previous research has suggested that extended exposure to EM fields damages cells that are in some way related to the transport, reabsorption or storage of iron.
+The researchers therefore studied how cells involved in these activities located in the brain, the kidney and the liver were affected by EM exposure.
+
+![“DNA com GGN”]({{ site.url }}/images/2016-08-28-mouse-dna-magnetic-fields.jpg)
+
+> _“DNA com GGN”. [Image](https://commons.wikimedia.org/wiki/File:DNA_com_GGN.jpg) by Nogas1974 / [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/)_
+
+Three sets of mice were kept in identical conditions differing only in the EM fields inside their cages.
+For eight weeks, one group of 20 mice was exposed to 0.1&nbsp;mT and the second (also 20) to 1.0&nbsp;mT, while the control group of 16 had no EM exposure beyond that of the earth's magnetic field (which was below 50&nbsp;&mu;T in the experimental area).
+After eight weeks, ten mice from each group were sacrificed in accordance with the protocol established to prepare their cells for analysis.
+Two of the mice from the control group were not treated with the agents used to prepare the rest for analysis, so as to provide background values for the measurements that were conducted.
+
+The cells were then given to team members for analysis without informing them which group of mice the cells came from.
```

Comment:
The researchers performed a so-called blind analysis, which means they didn't know which mice the cells came from, so that they didn't bias the results in any way with knowledge about the radiation levels involved.